### DISCUSSION TITLE
Show enabled value in command line

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1576] [Ideas]
It would be nice to see the enabled value for a setting that has fixed values such as `Change sound on click...` which has the values `off`, `1`, `2`, `3`, and `4` like having the enabled value be in the main theme color.

**Comment by Miodec** [comment]
What about when single command line mode is enabled?

What if some users might not understand instinctively that the highlight means 'currently active' and might get confused?

**Reply by unrealapex** [reply]
That is a valid point, maybe having another indicator of the value enabled?

**Reply by Miodec** [reply]
Also what about `Toggle` settings. How do you show the `true/false` state there.

**Reply by unrealapex** [reply]
Similar to the current theme enabled in settings how about a dot next to the enabled value? When the user hovers over the dot, a tooltip could be shown giving a better indication.
![Screenshot 2021-06-25 5 21 54 PM](https://user-images.githubusercontent.com/79121360/123495804-119caf00-d5da-11eb-9f56-d39f77a98ec2.png)

**Reply by Miodec** [reply]
This got me thinking maybe remove the `toggle` options entirely, and replace it with a submenu. And then to show the active state with an icon or something (maybe like tags?). It is one more click to for example disable punctuation, but at least its not ambiguous as to the current state of the punctuation.

**Reply by Miodec** [reply]
![image](https://user-images.githubusercontent.com/13181393/123495901-7a594b00-d61d-11eb-98b3-6c371b8f537c.png)
![image](https://user-images.githubusercontent.com/13181393/123495911-7e856880-d61d-11eb-8089-bc81b29a0b29.png)

Maybe we could use these

**Reply by unrealapex** [reply]
> This got me thinking maybe remove the `toggle` options entirely, and replace it with a submenu. And then to show the active state with an icon or something (maybe like tags?). It is one more click to for example disable punctuation, but at least its not ambiguous as to the current state of the punctuation.

Sounds good.

**Reply by unrealapex** [reply]
> 
> Maybe we could use these

They look pretty good but what color would the active state icon be? I'm thinking something like this. Using the main color seems optimal, but there could be better options.
```css
.enabled {
   color: var(--main-color);
{
```
